core/src/main/scala/org/apache/spark/errors/SparkCoreErrors.scala

```diff
@@ -467,6 +467,15 @@ private[spark] object SparkCoreErrors {
         "receivedBytes" -> receivedBytes.toString).asJava)
   }
 
+  def failedRenameTempFileError(srcFile: File, dstFile: File): Throwable = {
+    new SparkException(
+      errorClass = "_LEGACY_ERROR_TEMP_2199",
```

Comment:
Can't you re-using the existing one `FAILED_RENAME_PATH`?

Reply:
`FAILED_RENAME_PATH` is for the case when the destination already exists. `_LEGACY_ERROR_TEMP_2199` suits our case better.

Reply:
Can you assign a proper name?

Reply:
Assigned a name to the error class.